ProcessMaker/processmaker
# FOUR-19813 - Add Async Install for Individual Assets

ci:deploy

## Issue & Reproduction Steps
DevLink - Add Async Install for Individual Assets

## How to Test
Install a remote asset and make sure the progress bar displays

![image](https://github.com/user-attachments/assets/c2bc8790-f71c-4156-a9c0-d3b1c9529a82)


## Related Tickets & Packages
- https://processmaker.atlassian.net/browse/FOUR-19813

## Code Review Checklist
.
